src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  const UniqueValues = new Set();
+  const Values = Object.values(robot);
+
+  Values.forEach((elem) => {
+    UniqueValues.add(elem);
+  });
```

Comment:
no need to overcomplicate your code, it's possible to pass array to `Set()` to make only unique values stay in there